rustls/src/server/tls12.rs

```diff
@@ -173,19 +173,6 @@ mod client_hello {
                 ));
             }
 
-            let ecpoint = ECPointFormat::SUPPORTED
-                .iter()
-                .find(|format| ecpoints_ext.contains(format))
-                .cloned()
-                .ok_or_else(|| {
-                    cx.common.send_fatal_alert(
```

Comment:
`NoEcPointFormatsInCommon` doesn't seem to reappear in this diff, is that a problem?